app/views/projects/show.html.erb

```diff
@@ -181,8 +181,8 @@
                     </td>
                     <td>
                       <div class="ds_selector" >
-                        <label id="lbl_<%=s.id%>" class="mdl-checkbox mdl-js-checkbox mdl-js-ripple-effect pull-right" for="ds_<%=s.id%>">
-                          <input type="checkbox" id="ds_<%=s.id%>" class="mdl-checkbox__input <%if s.user_id == current_user.try(:id)%> mine<%end%>" checked>
```

Comment:
Removing this line breaks the "delete selected" button - needs to be brought back